ScoutSuite/providers/azure/resources/securitycenter/auto_provisioning_settings.py

```diff
@@ -0,0 +1,20 @@
+from ScoutSuite.providers.base.configs.resources import Resources
+
+
+class AutoProvisioningSettings(Resources):
+
+    def __init__(self, facade):
+        self.facade = facade
+
+    async def fetch_all(self):
+        for raw_settings in await self.facade.get_auto_provisioning_settings():
+            id, auto_provisioning_settings = self._parse(raw_settings)
```

Comment:
I think we decided to use the long name instead of just `_parse`.

```suggestion
            id, auto_provisioning_settings = self._parse_auto_provisioning_settings(raw_settings)
```

Reply:
Yeah please do, stuff like `_parse` is a bit cryptic.